src/fragments/lib/auth/common/mfa_preferences/common.mdx

```diff
@@ -0,0 +1,37 @@
+<Callout>
+Fetching and Updating MFA preferences is only available through the escape hatch APIs under the AWS Cognito Auth plugin. 
+</Callout>
+
```

Comment:
can we add an intro here about what TOTP is, and why a customer would use it before the callout.

You can use Time-based One-Time Password (TOTP) for multi-factor authentication (MFA) in your web or mobile applications. Amplify's Auth category includes support for handling TOTP setup and codes, offering an integrated solution for enhanced security for your users. Authenticator apps generate the temporary, one-time passwords. These apps, such as Google Authenticator, Microsoft Authenticator, have the TOTP algorithm built-in and work by using a shared secret key and the current time to generate a short-lived password.

Reply:
Done.